### ISSUE TITLE
Feature Request: Sectograph

### ISSUE THREAD

**Issue by Welding-Torch** [lynchjames/obsidian-day-planner#150]
<img align="center" src="https://user-images.githubusercontent.com/46340124/157181191-25609396-0975-43ce-a4c5-32de79d76a2b.png" width="619" height="720">

A **_Sectograph_** is a visual day planner that displays your calendar events on a pie chart in the form of a 12/24 hour clock face.

I think including a Sectograph view in the obsidian-day-planner plug-in is a **GOLDMINE** of an idea. It is a fairly rare graph, and thus it adds a lot of unique value to the extension. It is also just a REALLY good way to plan out your day and see what's happening when.

Do get back to me on if you're going to add this feature, I think it would be a really good idea.

♥Thank you!

---

**Some More descriptive images:**
![image](https://user-images.githubusercontent.com/46340124/157181248-e63172a2-d2cc-48df-9836-7b06abe6004d.png)
![image](https://user-images.githubusercontent.com/46340124/157181323-c69e03eb-3934-49ea-b9df-9694eb372ffc.png)

**Comment by lukemt** [comment]
pretty cool idea

**Comment by YeshuaWB3** [comment]
I completely agree. Fantastic 'at a glance' visual reference that would be infinitely useful in conjunction with Obsidian. At this point it's already connected by proxy through the calendar system but I believe a lot more could be done if there was a plug-in for it. As a matter of fact if I do have some time myself I just may look into building one. But I'm no professional and I really have no experience so I hope somebody else picks it up a real developer

**Comment by ivan-lednev** [comment]
Not planning to implement this. However, if there is a brave soul willing to submit a PR, I'll happily merge it.

**Comment by Welding-Torch** [comment]
I'm just surprised it took you this long to reply, I put this issue up over a year ago.

**Comment by lukemt** [comment]
@Welding-Torch Did you notice the repo owner has changed?
